### PR TITLE
nixos/manual: add pkg to the lexical scope of opengl.extraPackages

### PR DESCRIPTION
**Pull request by Jake-Gillberg**
###### Motivation for this change
Code snippet does not work when just adding to a configuration without a supporting `with pkgs;`

###### Things done

Small documentation change

### CODE REVIEW

**Comment by Jake-Gillberg** [comment]
@danieldk added this snippet recently, (merged by @mweinelt)

**Comment by danieldk** [comment]
> @danieldk added this snippet recently, (merged by @mweinelt) #95086

Thanks :+1: . Good catch! Could you also update the other examples using `extraPackages` in the same chapter? Could you also change the commit message to something like:

```
nixos/manual: add pkg to the lexical scope of opengl.extraPackages
```

? Makes it easier to change changes to the manual specifically :).

File: nixos/doc/manual/configuration/gpu-accel.xml [hunk 1]
```diff
@@ -100,7 +100,7 @@ Platform Vendor      Advanced Micro Devices, Inc.</screen>
        support. For example, for Gen8 and later GPUs, the following
        configuration can be used:
 
-	      <programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = [
+	      <programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = with pkgs; [
   intel-compute-runtime
```
Comment (jonringer) [inline nixos/doc/manual/configuration/gpu-accel.xml:104]: with pkgs should probably be avoid if possible, for an example, we could probably do
```suggestion
	      <programlisting><xref linkend="opt-hardware.opengl.extraPackages"/> = [
  pkgs.intel-compute-runtime
```

**Review by jonringer — COMMENTED** [review]
makes sense to me, however, `with pkgs;` should probably be discouraged

**Comment by jonringer** [comment]
@Jake-Gillberg thanks for opening your first PR with nixpkgs :)

**Comment by jonringer** [comment]
Please follow [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) and [manual#submitting-changes-making-patches ](https://nixos.org/nixpkgs/manual/#submitting-changes-making-patches) and squash the fix-up commits.

This can be done without `git rebase -i` by doing:
```
git reset HEAD~1                    # move fix-up commits into unstaged
git add -- pkgs/                    # move changes into staged
git commit --amend --no-edit        # add changes to previous commit
git push ... ... --force            # modify current PR branch
```

However, `git rebase -i` is a more powerful alternative, I created a small video demonstrating it's use [here](https://www.youtube.com/watch?v=OR8Q-4PLalc). A more indepth text tutorial can be found [here](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase)

**Pull request closed without merging**